testsuite/ui/views/master/login.py

```diff
@@ -0,0 +1,32 @@
+""" Representation of Login specific MASTER views"""
+from widgetastic.widget import View, Text
+
+from testsuite.ui.navigation import Navigable
+from testsuite.ui.views.admin.login import LoginWidget
+
+
+class MasterLoginView(View, Navigable):
```

Comment:
It seems to me this Login view shares more with admin `LoginView` than just login widget, wouldn't it make more sense to have common BaseLoginView and both admin/master extending it? @jsmolar @dhlavac WDYT?

Reply:
We agreed that we don't want to leverage inheritance between admin and master pages.

Reply:
@dlaso99 remove `Navigable`. Login view should not be navigated to. It is wrongly used in the admin login view as well.